frontend/src/pages/FilteredRecipes.js

```diff
@@ -2,11 +2,11 @@ import Main from "../components/Main";
 import RecipeList from "../components/RecipeList";
 import styled from "styled-components/macro";
 
-export default function FilteredRecipes({ filteredRecipes }) {
+export default function FilteredRecipes({ filteredRecipes, filteredLoading}) {
```

Comment:
Just a small tip for the future: when you pass down all the props you can make use of the spread operator:
```javascript
export default function FilteredRecipes(props) {
  return (
    <Main>
      <Titel>Mögliche Rezepte:</Titel>
      <RecipeList {...props} />
    </Main>
  );
}
```